localstack-core/localstack/services/sqs/provider.py

```diff
@@ -652,20 +690,14 @@ def list_messages(self, request: Request) -> ReceiveMessageResult:
         return self._get_and_serialize_messages(request, region, account_id, queue_name)
 
     @route("/_aws/sqs/messages/<region>/<account_id>/<queue_name>")
```

Comment:
q: not sure if it's normal that this method accept all HTTP methods?

Reply:
Perhaps we make this the same as the `/_aws/sqs/messages` route?
```suggestion
    @route("/_aws/sqs/messages/<region>/<account_id>/<queue_name>", methods=["GET", "POST"])
```